sdk/python/kfp/components/_data_passing.py

```diff
@@ -22,10 +22,11 @@
 
 
 import inspect
-import re
 from typing import Any, Callable, NamedTuple, Sequence
 import warnings
 
+from kfp.components import type_annotation_utils
```

Comment:
Can you please use relative imports here?  `from . import type_annotation_utils`

Reply:
Actually, per Google Python guide: "Do not use relative names in imports. Even if the module is in the same package, use the full package name. This helps prevent unintentionally importing a package twice."

https://google.github.io/styleguide/pyguide.html#224-decision